src/sentry/integrations/slack/utils.py

```diff
@@ -186,6 +186,15 @@ def build_attachment(group, event=None, identity=None, actions=None):
         color = ACTIONED_ISSUE_COLOR
         payload_actions = []
 
+    footer = u'{}'.format(group.qualified_short_id)
+
+    if rules:
+        rule = rules.pop(0)
```

Comment:
Personally, I'd try and avoid implicitly mutating argument values like this. It seems safe in this context but not very future-proof.

Reply:
Good point, I agree: 98bb14d